src/components/Tabs/Tabs.spec.tsx

```diff
@@ -0,0 +1,126 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import React, { useState } from "react";
+import { Tabs, TabSize, TabsPaddingX } from "@components/Tabs/Tabs";
+import { IconIcons, IconSize } from "@foundation/Icon";
+import { Card } from "@components/Card";
+import { BadgeStyle } from "@components/Badge";
+import { Button } from "@components/Button";
+import { TabItem, TabItemProps } from "@components/Tabs/TabItem";
+
+const data: TabItemProps[] = [
+    {
+        id: "tab-1",
+        label: "A tab",
+        children: "This is content for label 1",
+    },
+    {
+        id: "tab-2",
+        label: "Disabled",
+        disabled: true,
+        children: "This is content for label 2",
+    },
+    {
+        id: "tab-3",
+        label: "A long tab name",
+        decorator: <IconIcons size={IconSize.Size12} />,
+        children: "This is content for label 3",
+    },
+    {
+        id: "tab-4",
+        label: "An even longer tab name",
+        badge: { style: BadgeStyle.Positive, children: "Badge 1" },
+        children: (
+            <div>
+                <h2>Hello</h2>
+                <Card hoverable={false}>
+                    <p>This is content for tab 4</p>
+                </Card>
+            </div>
+        ),
+    },
+    {
+        id: "tab-5",
+        label: "All disabled",
+        decorator: <IconIcons size={IconSize.Size12} />,
+        badge: { style: BadgeStyle.Danger, children: "Badge 2" },
+        disabled: true,
+        children: <Button>This is content for label 5</Button>,
+    },
+    {
+        id: "tab-6",
+        label: "Last tab",
+        decorator: <IconIcons size={IconSize.Size12} />,
+        badge: { style: BadgeStyle.Danger, children: "Badge 2" },
+        children: <Button>This is content for label 5</Button>,
+    },
+];
+
+const TabComponent = () => {
+    const [activeItemId, setActiveItemId] = useState(data[0].id);
+    return (
+        <Tabs
+            activeItemId={activeItemId}
+            onChange={(value) => setActiveItemId(value)}
+            paddingX={TabsPaddingX.Small}
+            size={TabSize.Small}
+        >
+            {data.map((item) => (
+                <TabItem
+                    id={item.id}
+                    key={item.id}
+                    label={item.label}
+                    disabled={item.disabled ?? false}
+                    decorator={item.decorator}
+                    badge={item.badge}
+                >
+                    {item.children}
+                </TabItem>
+            ))}
+        </Tabs>
+    );
+};
+
+describe("Tabs Component", () => {
```

Comment:
We can add a few tests like:
- keyboard navigation
- testing accessibility and focus in the content area with focusable element (button, ...)
- should not be able to click on disabled tab (and it shouldn't emit event if clicked)
- should not pass over disabled tab when navigating with keyboard
- test keyboard accessibility in the overflow menu

Reply:
I do have a question though as I can't reach you out on slack (it's down). I noticed that the keyboard navigation only works with `tab` and `enter` and not with `ArrowLeft` or `ArrowRight`. Should I add the possibility to use those keys?